packages/frontend/src/routes/TransferWizardWrapper.js

```diff
@@ -0,0 +1,204 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Container from '../components/common/styled/Container.css'; 
+import cleanUpKeysImg from '../images/wallet-migration/screenshots/clean-up-keys.png';
+import logOutImg from '../images/wallet-migration/screenshots/log-out.png';
+import secureAccountsImg from '../images/wallet-migration/screenshots/secure-accounts.png';
+import transferAccountsImg from '../images/wallet-migration/screenshots/transfer-your-accounts.png';
+
+const Table = styled.table`
+    table-layout: fixed;
+    width: 100%;
+`;
+
+const Td = styled.td`
+    border-bottom: solid 1px lightgrey;
+    width: 100%;
+`;
+
+const ImageTd = styled(Td)`
+    text-align: center;
+`;
+
+const Warning = styled.p`
+    color: #980000;
+`;
+
+const Important = styled.p`
+    color: #ff0000;
+`;
+
+const ScreenshotImg = styled.img`
+    width: 300px;
+`;
+
+
+export const TransferWizardWrapper = () => {
+    return (
+        <Container>
+            <h6>The following should be devised as a page on the wallet.near.org site to inform users about TW.</h6>
+            <h1 >Migrating from the Near Wallet</h1>
```

Comment:
```suggestion
            <h1>Migrating from the Near Wallet</h1>
```

Also should it be just "Near Wallet" instead of "the Near Wallet"?